src/ec/system76/ec/system76_ec.c

```diff
@@ -2,8 +2,11 @@
 
 #include <bootstate.h>
 #include <arch/io.h>
+#include <cbfs.h>
 #include <console/system76_ec.h>
+#include <lib.h>
```

Comment:
probably not needed anymore, needed it for hexdump

Reply:
most likely

Reply:
Removed